src/app/settings/settings.page.ts

```diff
@@ -0,0 +1,84 @@
+import { Component, OnInit } from '@angular/core';
+import { ModalController, ToastController } from '@ionic/angular';
+import { UserService, Preferences, VoiceActor, VoiceActorService, PresentationOrder } from 'wanikani-api-ng';
+import { Observable } from 'rxjs';
+import { map, tap, take } from 'rxjs/operators';
+import { FormGroup, FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'app-settings',
+  templateUrl: './settings.page.html',
+  styleUrls: ['./settings.page.scss'],
+})
+export class SettingsPage implements OnInit {
+  preferences: Observable<Preferences>
```

Comment:
I don't think we need to keep a reference to the preferences observable

Reply:
removed